source/a_release_25062019.rst

```diff
@@ -4,6 +4,7 @@
 
 .. role:: blue
 
+========================
```

Comment:
Volgens mij zie ik al vier verschillende niveau's van titels in dit document. Zelfs in m'n proefschrift kon ik met drie niveau's toe :-)

In de README ook even duidelijk aangeven wat/wanneer qua headers.

Reply:
Op 1 plek kom ik tot 5 niveau's :(

Ben er ook niet helemaal blij mee, maar voor nu is het even goed. Zodra er een paar andere deadlines gehaald zijn qua projecten eens kijken of ik de boel kan versimpelen qua niveau's.